modules/cluster-logging-rn-5.4.2.adoc

```diff
@@ -0,0 +1,64 @@
+[id="cluster-logging-release-notes-5-4-2"]
+= Logging 5.4.2
```

Comment:
Same as 5.2.11 and 5.3.8. The bug list are updated close to the end. Please update the doc accordly.

Reply:
@anpingli - the following issues are not public and not included on the errata. Including them in the release notes would potentially create confusion.   
https://issues.redhat.com/browse/LOG-2672
https://issues.redhat.com/browse/LOG-2671
https://issues.redhat.com/browse/LOG-2670